deterministic-build.yml

```diff
@@ -0,0 +1,25 @@
+trigger:
+- master
+
+pool:
+  vmImage: windows-latest
+
+steps:
+- task: UseDotNet@2
+  displayName: 'Install .NET Core 3'
+  inputs:
+    version: 3.1.x
+
+- task: PowerShell@2
+  displayName: 'Build and compare'
+  inputs:
+    targetType: 'inline'
+    script: |
+      # Change git output from stderr to stdout https://github.com/dahlbyk/posh-git/issues/109.
+      $env:GIT_REDIRECT_STDERR = '2>&1'
+      dotnet run -p .\WalletWasabi.Packager\ --onlybinaries
+      Remove-Item -Recurse -ErrorAction SilentlyContinue .\WalletWasabi.Gui\bin\dist1
+      Rename-Item -Path .\WalletWasabi.Gui\bin\dist -NewName dist1
```

Comment:
I guess, it would be actually better to add a switch to Wasabi Packager that would affect destination folder. Something like:

`dotnet run -p .\WalletWasabi.Packager\ --onlybinaries --destination <folderHere>`